### PR TITLE
Only set CMAKE_CXX_STANDARD if not defined already

### PR DESCRIPTION
**Pull request by phcerdan**
This allow users to compile with their own standard.

# Checklist

- [NA] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [NA] Doxygen documentation of the code completed (classes, methods, types, members...)
- [NA] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: cmake/Common.cmake [hunk 1]
```diff
@@ -31,10 +31,14 @@ endforeach()
 
 
 # -----------------------------------------------------------------------------
-# CPP11 
+# CPP11
 # -----------------------------------------------------------------------------
-set (CMAKE_CXX_STANDARD 11)
-set (CMAKE_CXX_STANDARD_REQUIRED TRUE)
+if(NOT CMAKE_CXX_STANDARD)
```
Comment (dcoeurjo) [inline cmake/Common.cmake:36]: is CMAKE_CXX_STANDARD always >= 11 ?

**Review by dcoeurjo — COMMENTED** [review]
Thx @phcerdan fir the fix, just a minor question..

File: cmake/Common.cmake (hunk 1 above)
Comment (phcerdan) [reply cmake/Common.cmake:36]: Well, the user could explicitly provide `-DCMAKE_CXX_STANDARD=98`, bu t I have never seen that.
I can throw a fatal error if defined and less than 11, will update.

**Review by dcoeurjo — APPROVED** [review]

**Comment by dcoeurjo** [comment]
Perfect, thanks @phcerdan

**Pull request merged**